lua/autorun/server/sv_RoleVote.lua

```diff
@@ -76,7 +76,7 @@ function RoleVote:Start(time)
 	if not RoleVote.started then
 		votes = {}
 		RoleVote.started = true
-		time = time and time or 20
+		time = time and time or GetConVar("ttt_rolevote_votetime"):GetInt()
```

Comment:
The vote should start after mapchange with infinite time and automatically end when the first round begins, if it does not thats a bug but its working for me. So the convar seems useless to me.
You can see this with the countdown in the title of the RoleVote window, thats being displayed when using `ulx rv_start` but missing after mapchange.

![image](https://github.com/Blaubeeree/ttt2-rolevote/assets/47636067/1d12e279-44da-451d-80f4-7d37d73abac2)

Reply:
Yep doesn't make any sense..
Removed the commits.